tensorflow-notebook/Dockerfile

```diff
@@ -7,6 +7,6 @@ LABEL maintainer="Jupyter Project <jupyter@googlegroups.com>"
 
 # Install Tensorflow
 RUN pip install --quiet \
```

Comment:
Could you please add the `--no-cache-dir` option in the `pip install` command.

```diff
- RUN pip install --quiet \
+ RUN pip install --quiet --no-cache-dir \
```
Many thanks